numba/tests/test_np_functions.py

```diff
@@ -689,6 +721,116 @@ def test_diff2_exceptions(self):
                 cfunc(arr, n)
             self.assertIn("order must be non-negative", str(raises.exception))
 
+    def test_isclose(self):
+        def values():
+            yield 1e10, 1.00001e10, {}
+            yield 1e10, np.nan, {}
+            yield np.asarray([1e-8, 1e-7]), np.asarray([0.0, 0.0]), {}
+            yield np.asarray([1e10, 1e-7]), np.asarray([1.00001e10, 1e-8]), {}
+            yield np.asarray([1e10, 1e-8]), np.asarray([1.00001e10, 1e-9]), {}
+            yield np.asarray([1e10, 1e-8]), np.asarray([1.0001e10, 1e-9]), {}
+            yield np.asarray([1.0, np.nan]), np.asarray([1.0, np.nan]), {}
+            yield np.asarray([1.0, np.nan]),np.asarray([1.0, np.nan]),\
+                {'equal_nan':True}
+            yield np.asarray([np.nan, np.nan]), np.asarray([1.0, np.nan]),\
+                {'equal_nan':True}
+            yield np.asarray([1e-100, 1e-7]), np.asarray([0.0, 0.0]),\
+                {'atol':0.0}
+            yield np.asarray([1e-10, 1e-10]), np.asarray([1e-20, 0.0]), {}
+            yield np.asarray([1e-10, 1e-10]),\
+                np.asarray([1e-20, 0.999999e-10]),{'atol':0.0}
```

Comment:
Is `rtol` use tested?

Reply:
Included almost all tests that numpy uses to test `np.isclose`.